### DISCUSSION TITLE
vsCode Docker extention, how login with other user to Azure

### DISCUSSION THREAD

**Discussion by Hanswe** [microsoft/vscode-docker#3797] [Q&A] [answered]
The first time I used the Docker Extension in vs Code I was log in by Azure with the wrong user-id (non admin account) by Registries. I can't find how to logout and login with another user-id?
The only thing I can do is Disconnect from Azure by Registry, after this I do Connect Registry ans select Azure, than the old user-id is automatically used. There is no question for an user-id.

**Comment by bwateratmsft** [comment, marked as answer]
Hey @Hanswe, you'll want to run the "Azure: Sign Out" command provided by the Azure Account extension, followed by "Azure: Sign In", and then you can sign in to the correct account.

**Reply by Hanswe** [reply]
Het @bwateratmsft, thanks for your reply, how simple can it be :-). I thought extensions were completely separate.
